ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -151,6 +151,12 @@ def logspace(start, stop, num=50, endpoint=True, base=10.0, dtype=None, axis=0):
     return ivy.logspace(start, stop, num, base=base, axis=axis, dtype=dtype)
 
 
+@to_ivy_arrays_and_back
+def meshgrid(*x, sparse=False, indexing='xy'):
+    ivy_meshgrid = ivy.meshgrid(*x, sparse=sparse, indexing=indexing)
```

Comment:
`jax.numpy.meshgrid` has a `copy` argument, so it wouldn't be right to omit it. I would suggest you include it in the function's signature without handling it, and leave a ToDo comment saying it will be handled once `ivy.meshgrid` supports it.